setup.py

```diff
@@ -33,6 +33,19 @@
     'packages': pkgs,
     'package_dir': {'': 'src'},
     'package_data': {'pynwb':["%s/*.yaml" % schema_dir, "%s/*.json" % schema_dir]},
+    'classifiers': [
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3.6",
+        "Development Status :: 2 - Pre-Alpha",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Science/Research",
+        "Operating System :: Microsoft :: Windows",
+        "Operating System :: MacOS",
+        "Operating System :: Unix",
+        "Topic :: Scientific/Engineering :: Medical Science Apps."
```

Comment:
What do you think of adding `License :: OSI Approved :: BSD License` ?

https://pypi.python.org/pypi?:action=browse&c=60

Reply:
@jcfr I must have missed that one. I will add it now. Thanks !!!